multimodal-understanding/getting-started/06_tool_calling.ipynb

```diff
@@ -0,0 +1,349 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #59.    initial_response = client.converse(modelId=LITE_MODEL_ID, system=system, messages=messages, inferenceConfig=inf_params, additionalModelRequestFields={"inferenceConfig": {"topK":1}},toolConfig=tool_config)
</pre><p> </p><p>nit: repeated code. can we wrap them into a one function and just pass different tools?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-nova-samples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I don't think this would make it more repeatable. It's one line to call a function so it would just be a helper function to call a function with no additional logic (i.e. still functionally repeating the same code). In addition - this notebook allows you to call just one cell at a time which improves end user experience